versus-fe/src/index.js

```diff
@@ -5,11 +5,17 @@ import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 import { BrowserRouter } from 'react-router-dom';
 
-ReactDOM.render(
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import reducer from './redux/reducers';
+import logger from 'redux-logger';
+import apiMiddleware from './redux/middlewares/apiMiddleware';
 
-<BrowserRouter>
-  <App />
-</BrowserRouter>
+let Store = createStore(
```

Comment:
`const`

![](https://media.giphy.com/media/VBvt3V5fMHGZa/giphy.gif)

Damn! You're going to be strong!